### PR TITLE
update tests for HandlersTimeProfiler to fix occasional failures

### LINKED ISSUE #1824: Fix flaky test with time profiler
From time to time, tests related to time profiler are failing like there:
```
FAILED tests/ignite/contrib/handlers/test_time_profilers.py::test_pos_event_filter_threshold_handlers_profiler
= 1 failed, 702 passed, 29 skipped, 314 deselected, 134 warnings in 235.26s (0:03:55) =
```
https://github.com/pytorch/ignite/pull/1821/checks?check_run_id=2141981144

The idea is to inspect and analyse the failure case and propose a fix.

### PR DESCRIPTION
**Pull request by rushabh-v**
Fixes #1824

### CODE REVIEW

File: tests/ignite/contrib/handlers/test_time_profilers.py [hunk 1]
```diff
@@ -289,8 +289,10 @@ def delay_start(engine):
 
     assert "delay_start" in event_results[0]
     assert event_results[1] == "STARTED"
-
-    assert event_results[2] == approx(true_event_handler_time, abs=1e-1)  # total
+    if isinstance(event_results[2], str):
```
Comment (vfdev-5) [inline tests/ignite/contrib/handlers/test_time_profilers.py:292]: I would say this is strange as assertion as `STARTED` event should be always triggered if we run `dummy_trainer`.

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR @rushabh-v !
However, I'm not sure about the fix you are proposing. Yes, it could patch the flaky behaviour but it wont improve the feature of the tool.
Times used in the tests are greatly larger than the manual threshold used inside `HandlersTimeProfiler`, I think we have to inspect code a bit more...
What do you think ?

File: tests/ignite/contrib/handlers/test_time_profilers.py (hunk 1 above)
Comment (rushabh-v) [reply tests/ignite/contrib/handlers/test_time_profilers.py:292]: Okay sure, Let me take a deeper look to see if I find anything insightful.

**Comment by vfdev-5** [comment]
@rushabh-v I think it is very hardware related... Let's close this PR as I think there is no much point to investigate that. What do you think ?

**Comment by rushabh-v** [comment]
yes, I agree.

Closing this PR. Thanks!

**Pull request closed without merging**